VSDiagnostics/VSDiagnostics/VSDiagnostics/Utilities/Extensions.cs

```diff
@@ -10,9 +10,10 @@
 
 namespace VSDiagnostics.Utilities
 {
+    // TODO: after null checks, throw argumentnullexceptions instead of returning false
     public static class Extensions
     {
-        private static Dictionary<string, string> _aliasMapping = new Dictionary<string, string>
+        private static readonly Dictionary<string, string> AliasMapping = new Dictionary<string, string>
             {
                 {nameof(Int16), "short"},
```

Comment:
In fact, you should use Roslyn's http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Simplification/Simplifier.cs,161 instead of all of this.

That will also respect the user's settings.